src/azure-cli/azure/cli/command_modules/profile/_help.py

```diff
@@ -55,33 +77,54 @@
-long-summary: If no subscription is specified, shows the current subscription.
+long-summary: >
+    If no subscription is specified, information for the current subscription is returned.
+examples:
+    - name: >
+        Get the current default subscription returning results as JSON.  Use `--output yaml` for plain text. 
+        You can also get details of the current subscription by using `az account list --query "[?isDefault]"`.
+      text: >
+        az account show --output json
+    - name: Get information about a different subscription returning results as a table.
+      text: >
+        az account show --name "My subscription name" --output table
+    - name: Get information about a different subscription using the subscription ID.
+      text: >
+        az account show --name 00000000-0000-0000-0000-000000000000
+    - name: >
+        Get information about a subscription using `query`.  
+        To see available query options, first run `az account show --output json`.
+      text: >
+        az account show --query name
+        az account show --query user.name
+        az account show --query tenantId
+    - name: Return results in plain text.
+      text: >
+        az account show --query tenantId -o tsv
 """
 
 helps['account set'] = """
 type: command
-short-summary: Set a subscription to be the current active subscription.
-"""
-
-helps['account show'] = """
-type: command
-short-summary: Get the details of a subscription.
-long-summary: >-
-    If the subscription isn't specified, shows the details of the default subscription.
-
-
-    When --sdk-auth is used,
-    the output includes credentials that you must protect. Be sure that you do not include these credentials
-    in your code or check the credentials into your source control. As an alternative, consider using
-    [managed identities](https://aka.ms/azadsp-managed-identities) if available to avoid the need to use credentials.
+short-summary: Change the active subscription.
+examples:
+    - name: Change the active subscription using a subscription name
+      text: >
+        az account set --subscription "My subscription name"
+    - name: Change the active subscription using a subscription ID
+      text: >
+        az account set --subscription 00000000-0000-0000-0000-000000000000
+    - name: Change the active subscription using a variable
+      text: >
+        subscriptionId = "$(az account list --query "[?isDefault].id" -o tsv)"
+        az account set --subscription $subscriptionId
 """
 
 helps['account get-access-token'] = """
 type: command
 short-summary: Get a token for utilities to access Azure.
 long-summary: >
-    The token will be valid for at least 5 minutes with the maximum at 60 minutes.
-    If the subscription argument isn't specified, the current account is used.
+    The token will be valid for at least 5 minutes with a maximum of 60 minutes.
+    If the subscription parameter isn't specified, the current account is used.
```

Comment:
Even though we use word "parameter" and "argument" interchangeably, "argument" is the official word. For example, in the in-tool help:

```
> az group show -h

Command
    az group show : Gets a resource group.

Arguments
    --name --resource-group -g -n [Required] : Name of resource group. You can configure the default
                                               group using `az configure --defaults group=<name>`.

Global Arguments
    --debug                                  : Increase logging verbosity to show all debug logs.
    --help -h                                : Show this help message and exit.
    --only-show-errors                       : Only show errors, suppressing warnings.
    --output -o                              : Output format.  Allowed values: json, jsonc, none,
                                               table, tsv, yaml, yamlc.  Default: json.
    --query                                  : JMESPath query string. See http://jmespath.org/ for
                                               more information and examples.
    --subscription                           : Name or ID of subscription. You can configure the
                                               default subscription using `az account set -s
                                               NAME_OR_ID`.
    --verbose                                : Increase logging verbosity. Use --debug for full
                                               debug logs.
```

In Python, the library to parse arguments are called `argparse`, and the library to do auto-completion is called `argcomplete`.